### PR TITLE
mail-filter/rspamd: correct LuaBitOp dependency rule

### PR DESCRIPTION
**Pull request by arkamar**
Rspamd requires LuaBitOp dependency for all lua targets.

### CODE REVIEW

File: mail-filter/rspamd/rspamd-3.0-r1.ebuild [hunk 1]
```diff
@@ -30,7 +30,7 @@ REQUIRED_USE="${LUA_REQUIRED_USE}
 RDEPEND="${LUA_DEPS}
```
Comment (thesamesam) [inline mail-filter/rspamd/rspamd-3.0-r1.ebuild:30]: I'd revbump for such a change, but I can do this on merge.

**Comment by arkamar** [comment]
Thanks!

**Comment by Dessa** [comment]
are you sure thats correct? upstream doesn't include LuaBitOp in the luajit case, unless im missing something

**Comment by arkamar** [comment]
Yes, I am sure. `luajit` implements `bit` and this is the reason why upstream doesn't build it in again when `luajit` is used. However, the `LuaBitOp` package is done in such a way that it does not install any `bit.so` for `luajit`, therefore, it serves just like some sort of placeholder in that case and `luajit` will use its internal implementation anyway. I decided to change it this way because I wanted to have only one place where supported Lua versions are set.

**Pull request closed without merging**